vector/src/main/res/values/strings_tchap.xml

```diff
@@ -72,6 +72,9 @@
         <item quantity="other">History limited to %d days</item>
     </plurals>
     <string name="tchap_room_invited_you">%s invited you</string>
+    <string name="tchap_fab_new_direct_message">New direct message</string>
```

Comment:
Please use String ressource coming from Element `fab_menu_create_chat` instead of creating a new one and update the value to `New direct message`